session/container.go

```diff
@@ -63,49 +62,46 @@ func (x *Container) readContext(c session.TokenContext, checkFieldPresence bool)
 		return errors.New("container conflicts with wildcard flag")
 	}
 
-	verb := cCnr.Verb()
-	if verb > math.MaxInt32 {
-		return fmt.Errorf("verb %d overflows int32", verb)
+	verb := cCnr.GetVerb()
```

Comment:
why this check changed from overflows to negative?

Reply:
prev type `uint32`, now `int32`